dev/ci/coverage.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+yarn build
```

Comment:
Maybe `yarn build` is not needed for coverage.

Reply:
I tried without and it wouldn't work (some command wasn't found if I remember correctly)

Reply:
That's weird, `yarn build` only builds elm + stylus and pulls from transifex...

Reply:
OK. This issue could come from somewhere else. I've tried to remove it.